src/Angor/Client/Pages/Investor.razor

```diff
@@ -334,12 +372,12 @@
 
         var investorNostrPrivateKey = _DerivationOperations.DeriveProjectNostrInvestorPrivateKey(words, project.ProjectIdentifier);
         var investorNostrPubKey = investorNostrPrivateKey.PubKey.ToHex()[2..];
-        
+
         var investorProject = new InvestorProject
-        {
-            TransactionId = investment.TransactionId,
-            AmountInvested = investment.TotalAmount,
-        };
+            {
+                TransactionId = investment.TransactionId,
```

Comment:
I often see change just for spaces perhaps we should agree all of us on how we specify spaces, this is clutter for code reviewing